src/components/SubDescription.js

```diff
@@ -0,0 +1,191 @@
+import React, { useEffect, useState } from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+
+function SubProjectDescription({ selectedProject,issueNumber, currentIssue }) {
+  const [description, setDescription] = useState(null);
+  const { siteConfig } = useDocusaurusContext();
+  const API_BASE_URL = siteConfig.customFields.API_BASE_URL;
+  const API_AUTH_KEY = siteConfig.customFields.API_AUTH_KEY;
+
+  useEffect(() => {
+    fetch(
+      `${API_BASE_URL}/issues/${currentIssue}/${issueNumber}`,
+      {
+        method: "GET",
+        headers: {
+          "X-Secret-Key": API_AUTH_KEY,
+        },
+      },
+    )
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        setDescription(() => data);
+      })
+      .catch((error) => {
+        console.log("error", error.message);
+      });
+  }, [selectedProject]);
+
+  const parseMarkdown = (markdown) => {
+    markdown = markdown.replace(/^### (.*$)/gim, "<h3>$1</h3>");
```

Comment:
This doesn't seem clean. There should be a better way to do this. cc @amit-s19

Reply:
I was getting markdown responses and wasn't sure if the responses will be same like in md format I will try to implement other package to handle md